scripts/vim-patch.sh

```diff
@@ -87,7 +87,7 @@ commit_message() {
 }
 
 find_git_remote() {
-  git remote -v | awk '/neovim\/neovim.*(fetch)/{print $1}'
+  git remote -v | awk '$2 = /neovim\/neovim.*\(fetch\)$/ {print $1}'
```

Comment:
What's the `$2 = ` for?  That doesn't restrict the scope of the match.  It just overwrites the value of `$2`.

If you're trying just to match in `$2`, you would need `$2 ~` but `(fetch)` isn't in `$2`.